xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -1353,5 +1353,128 @@ workflow:
             architecture: x86_64</screen>
       <para></para>
     </sect2>
+
+    <sect2 xml:id="sec.obs.obs_scm_ci_workflow_integration.use_cases.create_package_releases_with_tags">
+      <title>Create Package Releases With Git Tags</title>
+
+      <para>Whenever you mark a software project release with a Git tag, you want to create a package
+      on OBS for this release. This is possible with the <emphasis>branch_package</emphasis> step and the
+      <emphasis>tag_push</emphasis> event filter.</para>
+
+      <para>After the usual <link linkend="sec.obs.obs_scm_ci_workflow_integration.setup">setup</link> for
+      OBS workflows with tokens and webhooks, you will need:</para>
+
+      <itemizedlist>
+        <listitem>
+          <para>A package in OBS that you own and for which you want to create releases, it will be the
+          <emphasis>source package</emphasis> inside the source project. e.g.:
+          <emphasis>home:jane/my_package</emphasis>.</para>
+        </listitem>
+
+        <listitem>
+          <para>A project in OBS that you own, it will be the <emphasis>target project</emphasis>. Let's say:
+          <emphasis>home:jane:releases</emphasis>.</para>
+        </listitem>
+
+        <listitem>
+          <para>A SCM repository for your software project with the source code and spec file in which you
+          will create Git tags to mark releases, e.g.: <emphasis>https://github.com/jane/my_package</emphasis>.</para>
+        </listitem>
+      </itemizedlist>
+
+      <para>The workflow configuration should be like this one:</para>
+
+      <screen>workflow:
+  steps:
+    - branch_package:
+        source_project: home:jane
+        source_package: my_package
+        target_project: home:jane:releases
+  filters:
+    event: tag_push</screen>
+
+      <note>
+        <para>A Git tag is a pointer to a commit, and commits exist independently of branches. This is why
+        filtering on branches is <emphasis>not</emphasis> possible when working with push events for tags.</para>
+      </note>
```

Comment:
Why mention this?

Reply:
I'm expecting questions about this. Should we remove this note?

Reply:
I think it won't be really common, so I'll remove it.